.eslintrc

```diff
@@ -28,375 +28,375 @@ globals:
+  sort-imports: [ 0 ]
+  sort-keys: [ 0 ]
+  sort-vars: [ 0 ]
+  space-before-blocks: [ 2, always ]
+  space-in-parens: [ 2, never ]
+  space-infix-ops: [ 2 ]
+  space-unary-ops: [ 2 ]
+  spaced-comment: [ 2, always ]
+  strict: [ 0 ]
+  switch-colon-spacing: [ 2 ]
+  symbol-description: [ 2 ]
+  template-curly-spacing: [ 2, never ]
+  template-tag-spacing: [ 2, never ]
+  unicode-bom: [ 2, never ]
+  unicorn/better-regex: [ 0 ]
+  unicorn/catch-error-name: [ 0 ]
+  unicorn/consistent-function-scoping: [ 2 ]
+  unicorn/custom-error-definition: [ 0 ]
+  unicorn/error-message: [ 0 ]
+  unicorn/escape-case: [ 0 ]
+  unicorn/expiring-todo-comments: [ 0 ]
+  unicorn/explicit-length-check: [ 0 ]
+  unicorn/filename-case: [ 0 ]
+  unicorn/import-index: [ 0 ]
+  unicorn/import-style: [ 0 ]
+  unicorn/new-for-builtins: [ 2 ]
+  unicorn/no-abusive-eslint-disable: [ 0 ]
+  unicorn/no-array-instanceof: [ 0 ]
+  unicorn/no-console-spaces: [ 0 ]
+  unicorn/no-fn-reference-in-iterator: [ 0 ]
+  unicorn/no-for-loop: [ 0 ]
+  unicorn/no-hex-escape: [ 0 ]
+  unicorn/no-keyword-prefix: [ 0 ]
+  unicorn/no-nested-ternary: [ 0 ]
+  unicorn/no-new-buffer: [ 0 ]
+  unicorn/no-null: [ 0 ]
+  unicorn/no-object-as-default-parameter: [ 2 ]
+  unicorn/no-process-exit: [ 0 ]
+  unicorn/no-reduce: [ 2 ]
+  unicorn/no-unreadable-array-destructuring: [ 0 ]
+  unicorn/no-unsafe-regex: [ 0 ]
+  unicorn/no-unused-properties: [ 2 ]
+  unicorn/no-useless-undefined: [ 0 ]
+  unicorn/no-zero-fractions: [ 2 ]
+  unicorn/number-literal-case: [ 0 ]
+  unicorn/numeric-separators-style: [ 0 ]
+  unicorn/prefer-add-event-listener: [ 2 ]
+  unicorn/prefer-array-find: [ 2 ]
+  unicorn/prefer-dataset: [ 2 ]
+  unicorn/prefer-event-key: [ 2 ]
+  unicorn/prefer-includes: [ 2 ]
+  unicorn/prefer-math-trunc: [ 2 ]
+  unicorn/prefer-modern-dom-apis: [ 0 ]
+  unicorn/prefer-negative-index: [ 2 ]
+  unicorn/prefer-node-append: [ 0 ]
+  unicorn/prefer-node-remove: [ 0 ]
+  unicorn/prefer-number-properties: [ 0 ]
+  unicorn/prefer-optional-catch-binding: [ 2 ]
+  unicorn/prefer-query-selector: [ 0 ]
+  unicorn/prefer-reflect-apply: [ 0 ]
+  unicorn/prefer-replace-all: [ 0 ]
+  unicorn/prefer-set-has: [ 0 ]
+  unicorn/prefer-spread: [ 0 ]
+  unicorn/prefer-starts-ends-with: [ 2 ]
+  unicorn/prefer-string-slice: [ 0 ]
+  unicorn/prefer-ternary: [ 0 ]
+  unicorn/prefer-text-content: [ 2 ]
+  unicorn/prefer-trim-start-end: [ 2 ]
+  unicorn/prefer-type-error: [ 0 ]
+  unicorn/prevent-abbreviations: [ 0 ]
+  unicorn/string-content: [ 0 ]
+  unicorn/throw-new-error: [ 2 ]
+  use-isnan: [ 2 ]
+  valid-typeof: [ 2, { requireStringLiterals: true } ]
+  vars-on-top: [ 0 ]
+  vue/attributes-order: [ 0 ]
+  vue/component-definition-name-casing: [ 0 ]
+  vue/html-closing-bracket-spacing: [ 0 ]
+  vue/max-attributes-per-line: [ 0 ]
+  vue/one-component-per-file: [ 0 ]
+  wrap-iife: [ 2, inside ]
+  wrap-regex: [ 0 ]
+  yield-star-spacing: [ 2, after ]
+  yoda: [ 2, never ]
```

Comment:
I don't particularily like this style of spacing and we don't do it in JS either. Maybe rename to `.eslintrc.yaml` to see if that shuts up your tool?

Reply:
@silverwind is `.eslintrc.yaml` supported by our js linter?

Reply:
never mind ... it does not work anyway ... - reverting

Reply:
`.eslintrc.yaml` should generally work except for maybe some editor integrations which is the reason I generally use the older `.eslintrc`.